packages/react/src/components/CircularProgressAvatar/CircularProgressAvatar.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {Badge, BadgeProps, CircularProgress, CircularProgressProps} from '@mui/material';
```

Comment:
Wrap these components. Even if not wrapped, always better to use the tree shakable import syntax.
https://mui.com/material-ui/guides/minimizing-bundle-size/#option-one-use-path-imports

Reply:
Got it. Will wrap and do this improvement in commits seperately.